frontend/shared-components/data-table/data-table.module.css

```diff
@@ -0,0 +1,27 @@
+.dataTable {
+  border-collapse: collapse;
+  margin-top: var(--size36);
+  width: 100vw;
+}
+
+.dataFooter,
+.dataHeader,
+.dataRows {
+  border-bottom: 3px solid black;
+  height: var(--size36);
+  text-align: center;
+}
+
+.dataFooter,
+.dataHeader {
+  background-color: var(--lightBlue);
+}
+
+.dataRows:nth-child(odd) {
+  background-color: rgba(124, 174, 215, 0.5);
```

Comment:
Hey! So sorry this was updated in the wireframes after you started this ticket, but do you mind updating this to `#B1CFE7` instead? We don't want the color to be transparent